src/index.tsx

```diff
@@ -0,0 +1,3 @@
+import Button from './components/Button/Button';
+
+export { Button };
```

Comment:
```suggestion
export * from "./components/Button"
```

also add index file in Button folder to reexport from it first

Reply:
Done